src/text_to_text_gen.py

```diff
@@ -0,0 +1,29 @@
+from transformers import pipeline, set_seed
+
+ttt_generator = None
+ttt_inited = False
+
+def init_ttt():
+    global ttt_generator
+    global ttt_inited
+
+    if ttt_inited:
+        return ttt_inited
+
+    try:
+        ttt_generator = pipeline('text-generation', model='gpt2')

```

Comment:
Why do you need a text generation model? Could you add a README file with the mini app description? If you are working with image captioning, then you could you only one model and extend its generation parameters. But maybe you came up with something more interesting.

Reply:
I updated README. In short, the first model generates a short description , so I uses the second one to expand it